cmd/utils/flags.go

```diff
@@ -1090,6 +1090,9 @@ func SetP2PConfig(ctx *cli.Context, cfg *p2p.Config) {
 	lightPeers := ctx.GlobalInt(LightLegacyPeersFlag.Name)
 	if ctx.GlobalIsSet(LightMaxPeersFlag.Name) {
 		lightPeers = ctx.GlobalInt(LightMaxPeersFlag.Name)
+	} else if lightClient && !lightServer && !ctx.GlobalIsSet(LightLegacyPeersFlag.Name) {
+		// dynamic default - for clients we use 1/10th of the default for servers
```

Comment:
Please simplify it by 

```go
// Recap the maximum light server number.
if lightClient && !ctx.GlobalIsSet(LightLegacyPeersFlag.Name) && !ctx.GlobalIsSet(LightMaxPeersFlag.Name) {
	lightPeers /= 10
}
```

Reply:
that's fine - I duplicated the `!lightServer` intentionally but I don't mind skipping it